src/rust/dbv2/check_db.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+echo "Starting a dedicated PostgreSQL Docker instance"
+
+docker run -d --rm --name econia_db_test -e POSTGRES_PASSWORD=pgpw -p 1984:5432 postgres > /dev/null
```

Comment:
So it turns out that `1984` is actually the TCP port for an Arweave mining node

https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers

And we of course wouldn't want to cause a port conflict with another crypto project

Is there a default test port we can use, or perhaps add in something that uses the first unassigned port?

Also, will this work if the user has an empty docker cache?